src/components/CohortBuilder/SaveVirtualStudiesModalContent.js

```diff
@@ -65,7 +69,7 @@ class SaveVirtualStudiesModalContent extends React.Component {
   }
 
   render() {
-    const { name, errorMessage } = this.state;
+    const { name, /*description,*/ errorMessage } = this.state;
```

Comment:
@jberube Why commented out?

Reply:
Because it's another feature, will be done separately.

I just did the plumbing work while I was at it.